cktap/base58.py

```diff
@@ -0,0 +1,96 @@
+import hashlib
```

Comment:
there probably is  a copyright notice needed here (from original)

Reply:
this is also from my repo, however I have copy pasted it from jimmy songs book - programming bitcoin - there is no licence in the file, but in README it says it is creative commons https://creativecommons.org/licenses/by-nc-nd/4.0/legalcode

I think as long as we have this opensource it is OK (imho, no expert in licensing)